cmd/tvx/main.go

```diff
@@ -0,0 +1,92 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+	"sort"
+	"strings"
+
+	"github.com/filecoin-project/go-jsonrpc"
+	"github.com/multiformats/go-multiaddr"
+	manet "github.com/multiformats/go-multiaddr/net"
+
+	"github.com/urfave/cli/v2"
+
+	"github.com/filecoin-project/lotus/api"
+	"github.com/filecoin-project/lotus/api/client"
+)
+
+var apiEndpoint string
+
+var apiFlag = cli.StringFlag{
+	Name:        "api",
+	Usage:       "json-rpc api endpoint, formatted as token:multiaddr",
```

Comment:
note also that all operations we're making against lotus are 'unprivileged', (just reading, not writing), so this can be safely specified as ":multiaddr" with no token provided.

Reply:
Nice, but that may change in the future. I'll surround the token in square brackets (denoting optionality), and will add a note.